sgv2-service-common/src/main/java/io/stargate/sgv2/common/cql/builder/QueryBuilderImpl.java

```diff
@@ -1523,6 +1523,14 @@ public static FunctionCall sum(String columnName, String alias) {
       return function(columnName, alias, "SUM");
     }
 
+    public static FunctionCall ttl(String columnName) {
+      return ttl(columnName, null);
+    }
+
+    public static FunctionCall ttl(String columnName, String alias) {
+      return function(columnName, alias, "TTL");
+    }
+
```

Comment:
@olim7t Can you help here and let me know if I need to change the DSL as well.. Seems that `ttl` was not included in the V1, most likely was missed.  See here:

```
      @SubExpr(
          name = "select",
          definedAs =
              "select star? column* function* ((count|min|max|avg|sum|writeTimeColumn) as?)* "
                  + "from (where* perPartitionLimit? limit? groupBy* orderBy*) allowFiltering?"),
```

Reply:
Other functions are there `(count|min|max|avg|sum|writeTimeColumn)`?

Reply:
Should I explore more in order to figure it out?

Reply:
Tried to play around with it, but basically needs more that adding a simple action there.. It's not important, but could be a nice addition in the future..